app/views/users/reactivate_profile/index.html.slim

```diff
@@ -4,7 +4,9 @@ h1.h3.my0 = t('forms.reactivate_profile.title')
 p.mt-tiny.mb0 = t('forms.reactivate_profile.instructions')
 = simple_form_for(@reactivate_profile_form, url: reactivate_profile_path,
   html: { autocomplete: 'off', method: :post, role: 'form' }) do |f|
+  - field_name = "#{f.object_name}[#{:recovery_code}][]"
   = f.error :base
-  = f.input :recovery_code, required: true
+  = f.error :recovery_code
+  = render 'partials/personal_key/entry_fields', field_name: field_name
```

Comment:
I would move this partial to the `shared` directory, such as `app/views/shared/_personal_key_fields.html.slim` or something, since we put a lot of shared partials there already